README.md

```diff
@@ -116,6 +116,35 @@ dependencies {
 }
 ```
 
+### Specifying a Checker Framework version
+
+To change the version of the Checker Framework used by the plugin (by default version 2.8.0),
+you should add `checkerFramework` entries to your `dependencies` block. 
+We recommend **explicitly** specifying the latest Checker Framework version, and updating
```

Comment:
Do we really recommend that?  It's the Gradle Way and the gradle developers do recommend it.
Maybe it would be clearer to note the possible consequences of not specifying it explicitly.